strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/test/java/org/carlspring/strongbox/storage/metadata/nuget/rss/PackageFeedTest.java

```diff
@@ -114,19 +78,20 @@ public void testUnmarshallFeed()
      * @throws Exception
      *             error during the test
      */
+    @ExtendWith({ RepositoryManagementTestExecutionListener.class,
+                  ArtifactManagementTestExecutionListener.class })
     @Test
-    public void testMarshallFeed()
+    public void testMarshallFeed(@NugetRepository(repositoryId = REPOSITORY_RELEASES)
+                                 Repository repository,
+                                 @NugetTestArtifact(repositoryId = REPOSITORY_RELEASES,
+                                                    id = "NUnit",
+                                                    versions = "2.5.9.10348")
+                                 Path artifactNupkgPath)
         throws Exception
     {
         // GIVEN
-        String packageId = "NUnit";
-        String packageVersion = "2.5.9.10348";
-        Path packageFilePath = TestCaseWithNugetArtifactGeneration.generateArtifactFile(baseDirectoryPath,
-                                                                                        packageId,
-                                                                                        packageVersion);
-
-        try (InputStream nupkgInputStream = new BufferedInputStream(Files.newInputStream(packageFilePath));
-                TempNupkgFile nupkgFile = new TempNupkgFile(nupkgInputStream);)
+        try (InputStream nupkgInputStream = new BufferedInputStream(Files.newInputStream(artifactNupkgPath));
+                TempNupkgFile nupkgFile = new TempNupkgFile(nupkgInputStream))
```

Comment:
Cosmetic: Please, re-indent to:
```
        try (InputStream nupkgInputStream = new BufferedInputStream(Files.newInputStream(artifactNupkgPath));
             TempNupkgFile nupkgFile = new TempNupkgFile(nupkgInputStream))
```

Thanks! :)

Reply:
Done!